Nuclei/Features/MissionService.cs

```diff
@@ -205,4 +208,66 @@ public static bool TryGetConsumePreselectedMission(out Mission? mission)
         PreselectedMissionKey = null;
         return true;
     }
+    
+    
+    // TODO: Deconstruct this into smaller, reusable functions
+    /// <summary>
+    ///     Starts the next mission in the mission rotation.
+    /// </summary>
+    public static async void StartNextMission(Player? player)
+    {
+        
+        try
+        {
+            var dsm = Globals.DedicatedServerManagerInstance;
+            if (dsm == null)
+            {
+                Nuclei.Logger?.LogWarning("dsm is null");
+                return;
+            }
+
+            var mr = dsm.missionRotation;
+            if (mr == null)
+            {
+                Nuclei.Logger?.LogWarning("missionRotation is null");
+                return;
+            }
+
+            var nextOpt = mr.GetNext();
+            if (!nextOpt.Key.TryGetKey(out var key))
+            {
+                Nuclei.Logger?.LogWarning("Error: could not resolve mission key.");
+                return;
+            }
+
+            if (!MissionSaveLoad.TryLoad(key, out var mission, out var err))
+            {
+                Nuclei.Logger?.LogWarning($"Load failed: {err}");
+                return;
+            }
+
+            Nuclei.Logger?.LogInfo($"Loading next mission: {mission?.Name ?? "<unnamed>"}");
+            if (player != null) ChatService.SendPrivateChatMessage("Loading next mission...", player);
+
+            // Switch to main thread for Unity scene/lobby ops
+            await UniTask.SwitchToMainThread();
+
+            dsm.UpdateLobby(mission, false);
```

Comment:
setting the 2nd argument to false seems to make it not show the time duration of the mission on the server list. When I set this to true, time shows up fine

Line 263 looks redundant if the argument in line 255 is set to true: `dsm.UpdateLobby(mission, true);`.